napari/components/dims.py

```diff
@@ -408,9 +585,62 @@ def _roll(self):
         order[nsteps > 1] = np.roll(order[nsteps > 1], 1)
         self.order = order.tolist()
 
+    def _sanitize_input(self, axis, value, value_is_sequence=False):
```

Comment:
As this coerces, a return type would be useful. I think it's `Tuple[List[int], List]` where the second list may store different types of values, but not sure.